src/chains.jl

```diff
@@ -637,25 +563,25 @@ function pool_chain(c::Chains)
     return Chains(pool_data, names(c), c.name_map; info=c.info)
 end
 
-function set_names(c::Chains, d::Dict; sorted::Bool=true)
-    # Set new parameter names.
-    params = names(c)
-    new_params = replace(params, d...)
+function replacenames(chains::Chains, old_new::Pair...)
```

Comment:
Yes, good idea, I thought about this at some point but forgot to add it! We could even think about dropping the `Dict` type annotation - then it would also support other types of dictionaries and named tuples.

Reply:
I've added the convenience function for AbstractDict.